app/assets/sass/application.scss

```diff
@@ -416,21 +416,30 @@
 
   .govuk-title {
     .context {
-      a {
-        color: $text-colour;
-        text-decoration: underline;
-
-        &:before {
-          content: " ";
-          display: inline-block;
-          height: 0;
-          width: 0;
-          border-top: 10px solid transparent;
-          border-bottom: 10px solid transparent;
-          border-right: 10px solid $black;
-          margin-right: 10px;
+      
+      display: none;
+      
+      @include media(mobile) {
```

Comment:
This only shows the "context" breadcrumb (above heading) on Mobile. `media(tablet)` shows it on tablet and above which is not what I wanted.